tutorial/practicals/nemo/3_nemo_openmp/omp_trans.py

```diff
@@ -60,8 +60,9 @@ def trans(psyir):
 
     :param psyir: the PSyIR of the provided file.
     :type psyir: :py:class:`psyclone.psyir.nodes.FileContainer`
-    '''
 
-    for loop in psyir.walk(Loop):
-        if loop.loop_type == "levels":
-            OMP_TRANS.apply(loop)
+    '''
+    routine = psyir.walk(Routine)[0]
+    for child in routine.children:
+        if isinstance(child, Loop) and child.loop_type == "levels":
+            OMP_TRANS.apply(child)
```

Comment:
That actually looks more complicated that the original loop. I might be missing the reason for this change (doesn't seem to correspond

Reply:
Yes, I'm unsure what to do here. The idea was to start off with something 'bad' and get the student to improve it as part of the tutorial. However, there's always the danger that people then assume that this is a 'good' starting point. I'll have another look.

Reply:
I decided to keep it as it is but with lots of warnings that it is deliberately not the way to go :-)

Reply:
OK, the comment certainly helped :)